.huly.nginx

```diff
@@ -118,6 +118,16 @@ server {
         proxy_pass http://stats:4900/;
     }
 
+    location /_telegram {
```

Comment:
Hi @IvanT771 
Thanks for your contribution.
I think Telegram should be an optional service.
I would suggest commenting out the configurations for it for now and adding a section to the README explaining how to enable it.
For example, this is how it's currently done for GitHub.